graph/tags.go

```diff
@@ -375,7 +390,7 @@ func (store *TagStore) GetRepoRefs() map[string][]string {
 	return reporefs
 }
 
-// Validate the name of a repository
+// ValidateRepoName validates the name of a repository.
```

Comment:
please fix the name of the function in the comment.
"ValidateRepoName" to "validateRepoName"